ccl/changefeedccl: TestParquetRows failed

ccl/changefeedccl.TestParquetRows [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/20766458?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/20766458?buildTab=artifacts#/) on master @ [f37e9c26d7d9affe6496172cdc647d49bd9e7cf7](https://github.com/cockroachdb/cockroach/commits/f37e9c26d7d9affe6496172cdc647d49bd9e7cf7):


```
=== RUN   TestParquetRows
    test_log_scope.go:171: test logs captured to: /artifacts/tmp/_tmp/caa72099da386999b346f93d4299a93a/logTestParquetRows2695337708
    test_log_scope.go:82: use -show-logs to present logs inline
    test_server_shim.go:155: automatically injected a shared process virtual cluster under test; see comment at top of test_server_shim.go for details.
    parquet_test.go:263: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/caa72099da386999b346f93d4299a93a/logTestParquetRows2695337708
--- FAIL: TestParquetRows (1.81s)
=== RUN   TestParquetRows/decimal
    row.go:35: 
        	Error Trace:	pkg/ccl/changefeedccl/cdctest/row.go:59
        	            				pkg/ccl/changefeedccl/cdctest/row.go:35
        	            				pkg/ccl/changefeedccl/parquet_test.go:269
        	            				pkg/ccl/changefeedccl/parquet_test.go:181
        	Error:      	Should be true
        	Test:       	TestParquetRows/decimal
--- FAIL: TestParquetRows/decimal (0.24s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/cdc
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestParquetRows.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-57166

ccl/changefeedccl.TestParquetRows [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/21071803?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/21071803?buildTab=artifacts#/) on master @ [7d464ada40b72ab7745d27ca3186951db7bc8e3b](https://github.com/cockroachdb/cockroach/commits/7d464ada40b72ab7745d27ca3186951db7bc8e3b):


```
=== RUN   TestParquetRows
    test_log_scope.go:172: test logs captured to: /artifacts/tmp/_tmp/1e20b8a3dad00d3d3a775716a087126a/logTestParquetRows2536368674
    test_log_scope.go:82: use -show-logs to present logs inline
    test_server_shim.go:183: automatically injected a shared process virtual cluster under test; see comment at top of test_server_shim.go for details.
    test_server_shim.go:679: DRPC is enabled (via testserver factory defaults)
    parquet_test.go:265: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/1e20b8a3dad00d3d3a775716a087126a/logTestParquetRows2536368674
--- FAIL: TestParquetRows (2.98s)
=== RUN   TestParquetRows/decimal
    row.go:35: 
        	Error Trace:	pkg/ccl/changefeedccl/cdctest/row.go:59
        	            				pkg/ccl/changefeedccl/cdctest/row.go:35
        	            				pkg/ccl/changefeedccl/parquet_test.go:271
        	            				pkg/ccl/changefeedccl/parquet_test.go:181
        	Error:      	Should be true
        	Test:       	TestParquetRows/decimal
--- FAIL: TestParquetRows/decimal (0.19s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestParquetRows.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

--- a/pkg/ccl/changefeedccl/parquet_test.go
+++ b/pkg/ccl/changefeedccl/parquet_test.go
@@ -20,6 +20,7 @@ import (
 	"github.com/cockroachdb/cockroach/pkg/ccl/changefeedccl/changefeedbase"
 	"github.com/cockroachdb/cockroach/pkg/jobs/jobspb"
 	"github.com/cockroachdb/cockroach/pkg/kv/kvpb"
+	"github.com/cockroachdb/cockroach/pkg/kv/kvserver"
 	"github.com/cockroachdb/cockroach/pkg/roachpb"
 	"github.com/cockroachdb/cockroach/pkg/sql"
 	"github.com/cockroachdb/cockroach/pkg/sql/sem/eval"
@@ -52,8 +53,9 @@ func TestParquetRows(t *testing.T) {
 	defer srv.Stopper().Stop(ctx)
 	s := srv.ApplicationLayer()
 
-	sysDB := sqlutils.MakeSQLRunner(srv.SystemLayer().SQLConn(t))
-	sysDB.Exec(t, "SET CLUSTER SETTING kv.rangefeed.enabled = true")
+	for _, l := range []serverutils.ApplicationLayerInterface{s, srv.SystemLayer()} {
+		kvserver.RangefeedEnabled.Override(ctx, &l.ClusterSettings().SV, true)
+	}
 
 	maxRowGroupSize := int64(4)
 

